Squablr/View Controllers/ProfileViewController.m

```diff
@@ -18,21 +19,59 @@ @implementation ProfileViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     // Do any additional setup after loading the view.
-    [self welcome];
+    [self queryUserProfileInfo];
 }
+- (IBAction)didTapEdit:(id)sender {
+    [self performSegueWithIdentifier:@"editSegue" sender:nil];
+}
+
 - (IBAction)didTapLogout:(id)sender {
     [GIDSignIn.sharedInstance signOut];
     [PFUser logOutInBackgroundWithBlock:^(NSError * _Nullable error) {
         if (!error) {
+            // Sends user to the login page
             UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
             LoginViewController *loginViewController = [storyboard instantiateViewControllerWithIdentifier:@"loginVC"];
             self.view.window.rootViewController = loginViewController;
         }
     }];
 }
 
--(void) welcome {
-    _welcomeMessage.text = [NSString stringWithFormat:@"%@%@%@", @"Hello, ", [PFUser currentUser].username, @"!"];
+-(void) queryUserProfileInfo {
```

Comment:
Think it might be a good idea to start putting all Parse query code in its own file. Remind me to discuss this with you, we can probably do this in a follow up PR.

Reply:
Will do